Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,277 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import requests
+import json
```

Comment:
My convention is to order the imports from the shortest to the longest line(the first 3 lines are not included)